imageresizerapp/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,5 @@
+Uninstall-ChocolateyPackage `
+  -PackageName 'imageresizerapp' `
+  -FileType 'EXE' `
+  -Silent '/quiet /norestart /uninstall' `
+  -File (((Get-UninstallRegistryKey 'Image Resizer for Windows').UninstallString -split '"')[1])
```

Comment:
If WiX registers everything with the MSI subsystem the same way and you can extract the product code, I'd go for it (with appropriate testing)! The MSI format can be decompiled, but I haven't seen the same for WiX.